mig/shared/compat.py

```diff
@@ -40,6 +40,14 @@
 _TYPE_UNICODE = type(u"")
 
 
+escape_html = None
+if PY2:
+    from cgi import escape as escape_html
+else:
+    from html import escape as escape_html
+assert escape_html is not None
```

Comment:
This assert looks redundant now.

Reply:
Yah - though what I am tempted to do actually is not centralise this function as part of this PR because it's adding unnecessary noise, and instead just add these lines inline then do a separate small fold things into compat change which would make what's happening here clearer I think.